vuu-ui/cypress/pages/ShellWithNewTheme.js

```diff
@@ -0,0 +1,31 @@
+import { SHELL_WITH_NEW_THEME_URL } from "../support/e2e/constants";
+
+export class ShellWithNewTheme {
+  visit() {
+    cy.visit(SHELL_WITH_NEW_THEME_URL);
+  }
+
+  getContextMenuButton() {
+    return cy
+      .findByRole("tablist", { name: "tab headers" })
+      .findAllByRole("tab")
+      .first()
+      .findByRole("button");
+  }
+
+  getSaveLayoutButton() {
+    return cy.findByRole("menuitem", { name: "Save Layout" });
+  }
+
+  getMyLayoutsButton() {
+    return cy.findByRole("tab", { name: "MY LAYOUTS" });
+  }
+
+  getSavedLayoutButton(layoutName, creator, date) {
+    const day = ("00" + date.getDate()).slice(-2);
+    const formattedDate = `${day}.${date.getMonth() + 1}.${date.getFullYear()}`;
+    const elementName = `${layoutName} ${creator}, ${formattedDate}`;
+
+    return cy.findByRole("button", { name: elementName });
+  }
```

Comment:
I feel like there must be a better way to do this. Suggestions welcome.

Reply:
I think here we can use formatDate from vuu-utils like such: `const formattedDate = formatDate(date, "dd.mm.yyyy")`

Reply:
In terms of a user point of view, I would imagine the ideal flow would be something more along the lines of what you have in `getContextMenuButton`. So `get all layouts in my layouts -> find group x, find the one with y name or z date`.

Reply:
@vferraro-scottlogic Thanks, that's exactly what I was looking for!

@cfisher-scottlogic I think that would involve a bit of refactor in the `LayoutList` component, but it should be doable. I'll look into it.